### PR TITLE
Feature/269 chapter project review date range

### PR DESCRIPTION
**Pull request by divideby0**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ source 'http://rubygems.org'
 
 ruby '2.1.7'
 
-gem 'rails', '3.2.22.3'
+gem 'rails', '3.2.22.4'
```
Comment (jcn) [inline Gemfile]: This change does not belong in this PR. Also, it looks like `3.2.22.5` is the most recent 3.2.x release (`3.2.22.4` looks like it introduced a regression).

Comment (divideby0) [reply Gemfile]: The upgrade to `3.2.22.4` was required to address this issue impacting the date selector:

https://github.com/justinfrench/formtastic/issues/1226

But I `3.2.22.5` should work just as well.

Comment (jcn) [reply Gemfile]: Totally understand; the upgrade to `3.2.22.5` should still have been in a new PR. 

That version bump did happen already though in https://github.com/awesomefoundation/awesomebits/pull/277 so you should be able to merge that into this PR and resubmit once the tests start passing.

**Comment by divideby0** [comment]
Deleted in favor of a cleaner WIP branch
